.github/workflows/npm_publish.yml

```diff
@@ -1,10 +1,9 @@
 name: NPM publish on new release
 
 on: 
-  release:
-    types: [created]
-    branches: 
-      - master
+  push:
```

Comment:
would it be possible to still have this on the `release` event, but with the tag filtering?

Reply:
No, that's not possible. I tried that initially, but it the filter would not take effect. I found in the docs that you can only filter the tags on push or the pull_request events

Reply:
ah ok, got it!